### PR TITLE
Fixed minor problem in searchField

### LINKED ISSUE #1674: Bug in the SearchField
**Actual Behaviour**

Even when the search box is empty, on clicking the up icon counter increases by one.

**Expected Behaviour**

When the search box is empty counter should always be at 0.

**Steps to reproduce it**

Go to https://chat.susi.ai/ and click on :mag:  icon and click on Up icon.

**Screenshots of the issue**

searchbug

**Would you like to work on the issue?**
Yes! :smile:

### PR DESCRIPTION
**Pull request by sunitasen**
Fixes #1674 

Changes: Updated the code so that whenever the searchField is empty counter remains at zero.

Demo Link: https://pr-1675-fossasia-susi-web-chat.surge.sh

### CODE REVIEW

File: src/components/ChatApp/SearchField.react.js [hunk 1]
```diff
@@ -140,14 +140,24 @@ class ExpandingSearchField extends Component {
...
           </span>
-          <IconButton
-            className="displayNone"
-            iconStyle={baseStyles.smallIcon}
-            style={baseStyles.icon}
-            onClick={this.onClickPrev}
-          >
-            <UpIcon />
-          </IconButton>
+          {this.props.searchText === '' ? (
```
Comment (AakashMallik) [inline src/components/ChatApp/SearchField.react.js]: Strings need to be in double quotes. Probably the reason why Codacy tests are failing.
```suggestion
          {this.props.searchText === "" ? (
```

**Comment by sunitasen** [comment]
Hey @akshatnitd your suggestion works perfectly fine, but it results in console error when the searchField is empty.

consoleerror

Should I commit the changes you suggested?

**Comment by AakashMallik** [comment]
Hey @sunitasen   , I was just going through this error and I realized that what we are doing here is basically preventing the bug from happening and not exactly curing it. I hope you get what I mean. Let this component be as it was and rather find out how searchIndex is increasing even with empty text. For that you'll have to trace back the component tree and find out the bug. Might be a tricky one.

**Comment by sunitasen** [comment]
Hey @AakashMallik  Thank You for pointing out where this **_onClickPrev** function exist! :smile: 
I was having hard time finding this, so came up with a simple solution! :sweat_smile:

**Comment by AakashMallik** [comment]
> Hey @AakashMallik Thank You for pointing out where this **_onClickPrev** function exist! smile
> I was having hard time finding this, so came up with a simple solution! sweat_smile

No problem.. :grinning:

**Comment by sunitasen** [comment]
@AakashMallik @anshumanv  I fixed **_onClickPrev** function. Please do have a look. :smile:

**Comment by AakashMallik** [comment]
@sunitasen Update your branch with master.

**Comment by sunitasen** [comment]
Hey @AakashMallik what do you mean by **log out** and **log back in**? Sorry, I am new here.

**Pull request merged**